multiraft/multiraft.go

```diff
@@ -388,6 +388,9 @@ type node struct {
 }
 
 func (n *node) registerGroup(groupID uint64) {
+	if groupID == noGroup {
+		panic("must not call registerGroup with noGroup")
+	}
```

Comment:
I wish we had something like panic(groupID == noGroup, "must...")
I see we don't but we should get some library that provides us with that because it's crazy to write these conditionals that will never execute.